homeassistant/components/matter/strings.json

```
@@ -324,6 +324,24 @@
       "battery_replacement_description": {
         "name": "Battery type"
       },
+      "battery_capacity": {
+        "name": "Battery capacity"
+      },
+      "battery_charge_state": {
+        "name": "Battery charge state",
+        "state": {
+          "charging": "Charging",
```

We have a common state for that one:

```suggestion
          "charging": "[%key:common::state::charging%]",
```